mobile/src/components/interview/components/applications/components/application-form/application-form.tsx

```diff
@@ -172,6 +178,19 @@ const ApplicationForm: FC<Props> = ({
           </View>
         </View>
       </View>
+      <View style={styles.rowData}>
+        <View style={styles.rowTitle}>
+          <Text style={styles.title}>Status</Text>
+        </View>
+        <View style={styles.rowContent}>
+          <Chip
+            text={interview.status}
+            color={
+              statusToColor[interview.status.toLowerCase() as InterviewStatus]
```

Comment:
Not sure if such type cast usage is correct

Reply:
Agree, why do we need toLowerCase here? `statusToColor` should have keys from `InterviewStatus`

Reply:
If remove toLowerCase, then the chip is not painted over. Back returns the status with a capital letter.

Reply:
isn't InterviewStatus  with cappital letter?

Reply:
Bug was local.